contrib/redhat/collectd.spec

```diff
@@ -810,11 +824,22 @@ Monitors process starts/stops via netlink library.
 Summary:	Python plugin for collectd
 Group:		System Environment/Daemons
 Requires:	%{name}%{?_isa} = %{version}-%{release}
-	%if 0%{?rhel} && 0%{?rhel} < 6
-BuildRequires: python26-devel
-	%else
+%if 0%{?rhel}
+%if 0%{?rhel} == 8
+BuildRequires: python3-devel
+%endif
+%if 0%{?rhel} == 7
+BuildRequires: python2-devel
+%endif
+%if 0%{?rhel} == 6
+BuildRequires: python2-devel
+%endif
+%if 0%{?rhel} <= 5
```

Comment:
RHEL 5 is eol so you can drop this. In that case you can reduce this down to python3-devel for rhel >= 8 and python2-devel for the rest

Reply:
When rebasing to master my CI is failing on all RHEL versions, so don't merge yet. (this used to work on 5.10 branch)